### ISSUE TITLE
AppCleaner: ACS: Further improve locale support on MIUI ROMs

### ISSUE THREAD

**Issue by JeffHorinzon** [d4rken/sdmaid-public#2816]
Boss (d4rken) told me that I can open an issue to further support more languages to come.  For now, WAVE 1 and 2 that were on #2802 are to be rolled out on v4.14.29.  This issue will be aimed to release on next version as of fix announced.

Instructions to help: Read the top of #2802

I'm aiming to cover ALL languages available in MIUI but any help would be greatly appreciated.
:)
Wave 3 and 4 will have languages that **REQUIRE** the attention of #334 , further notes to be discussed in the future.

**Comment by JeffHorinzon** [comment]
Here's WAVE 3, screenshots are arranging in progress...

Swedish
````
CD: Rensa cache?
Rensa data > Rensa cache > OK
Rensa cache > OK
````

Vietnamese (Note: This language heavily focuses on accents)
````
CD: Xóa bộ nhớ đệm?
Xóa dữ liệu > Xóa bộ nhớ đệm > OK
Xóa bộ nhớ đệm > OK
````

Greek
````
CD: Εκκαθάριση προσωρινή μνήμης;
Εκκαθάριση δεδομένων > Εκκαθάριση προσωρινή μνήμης > ΟΚ
Εκκαθάριση προσωρινή μνήμης > ΟΚ
````

Belarusian
````
CD: Ачысціць кэш?
Ачысціць дадзеныя > Ачысціць кэш > ОК
Ачысціць кэш > ОК
````

Bulgarian
````
CD: Изчисти кеша?
Изчисти данни > Изчисти кеша > ОК
Изчисти кеша > ОК
````

Kazakh
````
CD: Кеш тазалансын ба?
Деректерді жою > Кэшті тазалау > ОК
Кэшті тазалау > ОК
````

Macedonian 
````
CD: Да се избрише кеш меморијата?
Избриши податоци > Исчисти кеш меморија > ОК
Исчисти кеш меморија > ОК
````

Serbian
````
CD: Очисти кеш?
Избриши податке > Очисти кеш > У реду
Очисти кеш > У реду
````

Georgian
````
CD: გავწმინდო ქეში?
მონაცემების გასუფთავება > ქეშის გასუფთავება > OK
ქეშის გასუფთავება > OK
````

Armenian (NOTE: Some text editors display modifier ՞ apart from the letter ե but that’s ok, it’s natural)
````
CD: Մաքրե՞լ քեշը:
Մաքրել տվյալները > Մաքրել քեշը > OK
Մաքրել քեշը > OK
````

Languages here requires ACS RTL Layout support.  Refer to #334
Hebrew (WARNING: RTL SUPPORT REQUIRED) (Note: Yep, I wrote these myself, everything here is literally read RTL)
````
CD: ?לנקות מטמון
נקה נתונים > ניקוי מטמון > אישור
ניקוי מטמון > אישור
````

Bahraini Urdu (WARNING: ANOTHER RTL)
````
CD: كيشے صاف كريں؟
ڈیٹا صاف كريں > كيشے صاف كريں > ٹھیک ہے 
كيشے صاف كريں > ٹھیک ہے
````

-=Screenshots coming soon=-

**Comment by d4rken** [comment]
> Wave 3 and 4 will have languages that REQUIRE the attention of #334 , further notes to be discussed in the future.

I think this might just work. I'll send you a test version to confirm the RTL behavior.

**Comment by d4rken** [comment]
Greek is the only language where the dialog title doesn't end in q `?` or is that a typo?

**Comment by d4rken** [comment]
Well Amernian doesn't either, both not a typo?

**Comment by JeffHorinzon** [comment]
> Greek is the only language where the dialog title doesn't end in q `?` or is that a typo?
> 
> Well Armenian doesn't either, both not a typo?

They are not typos, they were literally `;` and `:`.
Greek
![Screenshot_2019-07-15-20-12-33-592_com miui securitycenter](https://user-images.githubusercontent.com/3746534/61227790-8eb91b80-a757-11e9-8a31-e5cc73b96ddd.png)
Armenian
![Screenshot_2019-07-15-21-08-51-267_com miui securitycenter](https://user-images.githubusercontent.com/3746534/61227818-a1335500-a757-11e9-84e6-bfa4c9f1fc07.png)

**Comment by JeffHorinzon** [comment]
Back with the last wave of languages!
This should cover the entire MIUI language set.  However, I couldn't type some down for either reasons:
1) Google Translate doesn't have camera capture for that language
2) Their keyboard was too complex to learn.
3) Google Translate DOES NOT have that language at all
If anyone knows how to type these missing languages down, please do help us by replying to this issue. (Screenshots coming soon, bear with me).

Without further ado, here's the list:

Pakistani Urdu (WARNING: ALSO RTL)
````
CD: كيشے صاف كريں؟
ڈیٹا صاف كريں > كيشے صاف كريں > ٹھیک ہے 
كيشے صاف كريں > ٹھیک ہے
````

Arabic (RTL)
````
CD: مسح الذاكرة المؤقتة ؟
مسح البيانات > مسح ذاكرة التخزين المؤقتة > موافق
مسح ذاكرة التخزين المؤقتة > موافق
````

Persian (RTL)
````
CD: حافظه پنهان پاک شود ؟
پاک كردن داده ها > پاک سازی حافظه پنهان > تأييد
پاک سازی حافظه پنهان > تأييد
````

Nepali (India)
````
CD: क्यास खाली गर्नुहुन्छ?
डाटा खाली गर्नुहोस् > क्यास खाली गर्नुहोस् > ठिक छ
क्यास खाली गर्नुहोस् > ठिक छ
````

Nepali (Nepal)
````
CD: क्यास खाली गर्नुहुन्छ?
डाटा खाली गर्नुहोस् > क्यास खाली गर्नुहोस् > ठिक छ
क्यास खाली गर्नुहोस् > ठिक छ
````

Marathi
````
CD: कॅचे पुसायची?
डेटा साफ करा > कॅचे पुसा > ओके
कॅचे पुसा > ओके
````

Assamese (Need attention: Written based on Bengali script, may be inaccurate)
````
CD: কেশ্ব পৰিষ্কাৰ কৰিবনে?
ডাটা পৰিষ্কাৰ কৰক > কেশ্ব পৰিষ্কাৰ কৰক > ঠিক
কেশ্ব পৰিষ্কাৰ কৰক > ঠিক
````

Bengali (India)
````
CD: ক্যাশে পরিষ্কার করবেন?
ডেটা পরিষ্কার করুন > ক্যাশে পরিষ্কার করুন > ঠিক আছে
ক্যাশে পরিষ্কার করুন > ঠিক আছে
````

Bengali (Bangladesh)
````
CD: ক্যাশে পরিষ্কার করবেন?
ডেটা পরিষ্কার করুন > ক্যাশে পরিষ্কার করুন > ঠিক আছে
ক্যাশে পরিষ্কার করুন > ঠিক আছে
````

Punjabi
````
CD: ਕੈਸ਼ੇ ਹਟਾਉਣੇ ਹਨ?
ਡਾਟਾ ਸਾਫ਼ ਕਰੋ > ਕੈਸ਼ੇ ਸਾਫ਼ ਕਰੋ > ਠੀਕ
ਕੈਸ਼ੇ ਸਾਫ਼ ਕਰੋ > ਠੀਕ
````

Gujarati
````
CD: કૅશ સાફ કરીએ?
ડેટા સાફ કરો > કૅશ સાફ કરો > ઑકે
કૅશ સાફ કરો > ઑકે
````
Odia (Incomplete)
````
CD:
ଡାଟା ଖାଲିକରନ୍ତୁ > 
````

Tamil
````
CD: தேக்ககத்தை அழிக்கவா?
தரவை அழி > தேக்ககத்தை அழி > சரி
தேக்ககத்தை அழி > சரி
````

Telugu
````
CD: కాష్ ను తీసివేయాలా?
డేటా తొలగించండి > కాష్ ని తొలగించు > సరే
కాష్ ని తొలగించు > సరే
````

Kannada 
````
CD: ಕ್ಯಾಶೆ ತೆರವುಗೊಳಿಸುವುದೇ?
ಡೇಟಾ ಅಳಿಸಿ > ಕ್ಯಾಶೆ ಅಳಿಸಿ > ಸರಿ
ಕ್ಯಾಶೆ ಅಳಿಸಿ > ಸರಿ
````

Malayalam
````
CD: കാഷേ മായ്ക്കണോ?
ഡാറ്റ മായ്ക്കുക > കാഷേ മായ്ക്കുക > ശരി
കാഷേ മായ്ക്കുക > ശരി
````

Thai
````
CD: ล้างหน่วยความจำแคช?
ล้างข้อมูล > ล้างหน่วยความจำแคช > ตกลง
ล้างหน่วยความจำแคช > ตกลง
````

Burmese (Note: Typed by hand, might have mistakes)(Incomplete, Couldn't find correct word)
````
CD: 
ဒေတာရှင်းပါ > 
````

Khmer (Incomplete, similar case for Burmese)
````
CD: 
````

**Comment by d4rken** [comment]
Partly included in v4.14.30 though not every locale is fully functional.

**Comment by d4rken** [comment]
Awesome work @JeffHorinzon
